components/combobox/index.css

```diff
@@ -0,0 +1,609 @@
+      border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-focus-indicator-color, var(--spectrum-combobox-focus-indicator-color)));
+      border-radius: calc(
+        var(--mod-combobox-border-radius, var(--spectrum-combobox-border-radius)) +
+        var(--mod-combobox-focus-indicator-gap, var(--spectrum-combobox-focus-indicator-gap)) +
+        var(--mod-combobox-border-width, var(--spectrum-combobox-border-width))
+      );
+    }
+  }
+}
+
+.spectrum-Combobox-input {
+  padding-block-start: var(--mod-combobox-spacing-block-start-edge-to-textfield, var(--spectrum-combobox-spacing-block-start-edge-to-text));
+  padding-block-end: var(--mod-combobox-spacing-block-end-edge-to-textfield, var(--spectrum-combobox-spacing-block-end-edge-to-text));
+  padding-inline-start: var(--mod-combobox-spacing-inline-start-edge-to-textfield, var(--spectrum-combobox-spacing-inline-start-edge-to-text));
+  padding-inline-end: calc(var(--spectrum-combobox-button-width) + var(--spectrum-combobox-spacing-inline-end-edge-to-text));
+
+  background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-default, var(--spectrum-combobox-background-color-default)));
+  border-color: var(--highcontrast-combobox-border-color, var(--mod-combobox-border-color-default, var(--spectrum-combobox-border-color-default)));
+  backface-visibility: hidden;
+  
+  font-size: var(--mod-combobox-font-size, var(--spectrum-combobox-font-size));
+  font-family: var(--mod-combobox-font-family, var(--spectrum-combobox-font-family));
+  font-weight: var(--mod-combobox-font-weight, var(--spectrum-combobox-font-weight));
+  line-height: var(--mod-combobox-line-height, var(--spectrum-combobox-line-height));
+  font-style: var(--mod-combobox-font-style, var(--spectrum-combobox-font-style));
+  color: var(--highcontrast-combobox-font-color, var(--mod-combobox-font-color-default, var(--spectrum-combobox-font-color-default)));
+
+  /* fill space */
+  flex: 1;
+
+  &::placeholder {
+    color: var(--highcontrast-combobox-font-color-placeholder, var(--mod-combobox-font-color-placeholder, var(--spectrum-combobox-font-color-placeholder)));
+  }
+
+  &:hover,
+  &:active {
+    background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-hover, var(--spectrum-combobox-background-color-hover)));
+    border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-hover, var(--spectrum-combobox-border-color-hover)));
+    color: var(--highcontrast-combobox-font-color, var(--mod-combobox-font-color-hover, var(--spectrum-combobox-font-color-hover)));
+
+    &::placeholder {
+      color: var(--highcontrast-combobox-font-color-placeholder, var(--mod-combobox-font-color-hover, var(--spectrum-combobox-font-color-hover)));
+    }
+  }
+
+  &:focus,
+  &.is-focused {
```

Comment:
Does the `.spectrum-Combobox-input` ever get an `is-focused` class? Maybe this should be `spectrum-Combobox-textfield.is-focused &`

Reply:
Yes, you're right. I don't think it's added to the input. Removing that in the next commit 👍